containers/geth/is-geth-synchronized.sh

```diff
@@ -0,0 +1,36 @@
+#! /bin/sh -e
+geth_address="$1"
+
+
+message="$(cat<<EOF
+{
+    "jsonrpc": "2.0",
+    "method":  "eth_syncing",
+    "params":  [],
+    "id":      4
+}
+EOF
+)"
+
+
+response="$(
+    wget                                                                            \
+        --output-document -                                                         \
+        --quiet                                                                     \
+        --header    "Content-Type:application/json"                                 \
+        --post-data "$message"                                                      \
+        "$geth_address"                                                             \
+)"
+
+synchronization_status="$(echo $response | jq '.result')"
+if [[ "$synchronization_status" == null ]]; then
```

Comment:
What if it's something else than `null`? In bash `null` is just a string. You could get some other string here. I mean, you probably shouldn't get anything other than `true`/`false` if the command is correct but we're talking about handling unexpected stuff after all.

I think the condition should look like this:

```
If true
    ...
if false
    ...
else
    ...
```

Reply:
In this case they are only three possibly value: true, false, null. The value is return by `jq` that parse specific json key. The bash only handle a result. Are you sure we want change the solution ?

Reply:
OK. I see the problem. There's no `true`. It returns a dict. In that case check for `false`, `null` and everything else.